src/components/HeaderForm.js

```diff
@@ -825,6 +836,15 @@ class HeaderForm extends Component {
     this.setState({ simplify: !this.state.simplify });
   };
 
+  /* Function for toggling display of node sequences */
+  toggleIncludeSequences = () => {
+    this.setState({ removeSequences: !this.state.removeSequences });
+  };
+
+  openPopup = () => {
```

Comment:
This really toggles it.
```suggestion
  togglePopup = () => {
```